### PR TITLE
Add base functionality

### PR DESCRIPTION
**Pull request by rogeruiz**
This adds the base functionality for `coming-attractions`. As of right now, it's mostly a work in progress and completely open to discussion. Comments welcome below! 🎉 

### Acceptance Criteria:

- [x] Make it Node.
    - [x] Write some integration tests.
    - [x] Write more complete integration tests.
- [ ] Talk a bit about how node is being used for the generation of the files, style maybe?
- [ ] Talk a bit about publishing releases to npm, using npm in this way.
- [x] Make it work.
- [x] Include an `.eslintrc` and `.editorconfig`, the latter with a JS focus.
- [ ] Talk a bit about integrating this into already existing `npm` automation workflows or apps at :18f: so you have a static app you can deploy at any time.

### CODE REVIEW

**Comment by rogeruiz** [comment]
:eyes: https://coming-attractions.apps.cloud.gov/anything/but/production

Currently deployed to my `gsa-sandbox`.

As I've considered this a collaboration between DevOps and Frontend, I'm pinging a person from each guild.
cc: @linuxbozo & @xtine

File: source/cf/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (adborden) [inline source/cf/manifest.yml]: Is there a suffix we can add to this file to indicate it's a template? Not sure if mustache has a preferred template extension but something like `manifest.yml.tpl` would be more descriptive.

Comment (rogeruiz) [reply source/cf/manifest.yml]: yeah, maybe `<file_name>.<extension>.mo` ?

**Comment by adborden** [comment]
:+1:  This could probably go in a bunch of different directions, I'm curious why not use one of the generator frameworks, like [yeoman](http://yeoman.io/)?.

**Comment by adborden** [comment]
IIRC, the front end guild standardized on the airbnb styleguide, should we just us the airbnb `.eslintrc`?

**Comment by rogeruiz** [comment]
@adborden mostly because of [this](https://github.com/18F/coming-attractions#development)

> In order to work on this project, it's recommended that the source code be maintained in a separate repository. It does not necessarily need to be a Github-type of fork. It can be a completely separate project with a name more appropriate for your needs.

I figured I'd just use mustache to not complicate the build process as other projects may either fork this repository to build their opinionated version or keep it as simple as possible.

File: source/cf/manifest.yml (hunk 1 above)
Comment (rogeruiz) [reply source/cf/manifest.yml]: ✅ https://github.com/18F/coming-attractions/pull/1/commits/09358a19412a35703c62fdfa3e4b33d127db1bf5

**Review by adborden — APPROVED** [review]

**Comment by rogeruiz** [comment]
@glbrtmrgn will be covering the following points in separate issues.
- Talk a bit about how node is being used for the generation of the files, style maybe?
- Talk a bit about publishing releases to npm, using npm in this way.
- Talk a bit about integrating this into already existing npm automation workflows or apps at :18f: so you have a static app you can deploy at any time.

**Review by glbrtmrgn — APPROVED** [review]
:shipit:

**Pull request merged**